src/core/src/sim/mc/moves/rotate.rs

```diff
@@ -154,3 +151,15 @@ fn rotation_matrix(axis: &Vector3D, angle: f64) -> Matrix3 {
 
     return rotation;
 }
+
+/// Returns a random unit vector,
+/// i.e. a random point on an unit sphere.
+/// @TODO: Maybe move this to types::Vector3D?
+fn random_unit_vector3d (rng: &mut Box<Rng>) -> Vector3D {
```

Comment:
The algorithm changed here. Why not use the nice 3D normal distribution => uniform distribution on the R3 sphere feature?